storage/src/test/java/tech/pegasys/teku/storage/server/VersionedDatabaseFactoryTest.java

```diff
@@ -63,7 +63,7 @@ public void createDatabase_fromExistingDataDir() throws Exception {
   public void createDatabase_asV4Database() throws Exception {
     final DatabaseFactory dbFactory =
         new VersionedDatabaseFactory(
-            new StubMetricsSystem(), dataDir.toString(), DATA_STORAGE_MODE, "4");
+            new StubMetricsSystem(), dataDir.toString(), DATA_STORAGE_MODE, "4", 1L);
```

Comment:
i wonder if i need a factory constructor that doesn't take this new parameter @ajsutton but it'd only really be in tests like this that its made use of...

Reply:
hmm, hard to say.  I don't think 1 is a particularly good default to embed so I'd probably leave it but wouldn't block it either way.

Reply:
It might make sense to have 1 as a default for some of the test factories we have (eg InMemoryStorageSystem) but I'm not sure if doing it at that level actually provides much benefit.